extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-thrift-server-chart/README.md

```diff
@@ -0,0 +1,37 @@
+# aiSSEMBLE&trade; Spark Thrift Helm Chart
+Baseline Helm chart for deploying the Spark Thrift Server. Chart is built and managed during the normal Maven build lifecycle and placed in the **target/helm/repo** directory. See https://github.com/kokuwaio/helm-maven-plugin for more details.
+
+# Basic usage with Helm CLI
+To use the module, perform [extension-helm setup](../README.md#leveraging-extensions-helm) and override the chart version with the desired aiSSEMBLE version. For example:
+```bash
+helm install thriftserver ghcr.io/boozallen/aissemble-thrift-server-chart --version <AISSEMBLE-VERSION>
+```
+_**NOTE:**_ *the version should match the aiSSEMBLE project version.*
+
+# Properties
+
+| Property                         | Description                                                                                                                       | Default                                                               |
+|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|
+| enable                           | Enable or disable the entirety of the spark-thrift-server deployment.  When false, equivalent to not installing the chart.        | true                                                                  |
+| deployment.annotations           | Annotations to apply to the Spark Thrift Server Deployment.                                                                       | {}                                                                    |
+| deployment.labels                | Labels to apply to the Spark Thrift Server Deployment.                                                                            | {}                                                                    |
+| deployment.replicas              | Number of replicas for the Spark Thrift Server Deployment.                                                                        | 1                                                                     |
+| deployment.image.repository      | Repository for the Spark Thrift Server image.                                                                                     | "apache/spark"                                                        |
+| deployment.image.tag             | Tag for the Spark Thrift Server image.                                                                                            | "3.5.1"                                                               |
+| deployment.image.imagePullPolicy | Image pull policy for the Spark Thrift Server image.                                                                              | "IfNotPresent"                                                        |
```

Comment:
A: Why not just `deployment.image.pullPolicy` or `deployment.imagePullPolicy`?

Reply:
This is consistent with most of our other aissemble charts-- The only real departure from our normal default is storing it under the `deployment` tag as opposed to having a standalone `image` section.